lib/engine/action/auto.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+# The Auto action type exists for auto_actions
+# This is to be used when the action that is automatically being done
+# is a basic check or otherwise not worth creating an Action class
+# Details is a string.
+module Engine
+  module Action
+    class Auto < Base
+      attr_reader :details
```

Comment:
I think I'd prefer this as a specific DestinationConnected with a list of corps

Reply:
I renamed it to DestinationConnection as it's vague enough to be reused elsewhere :)